heron/common/src/python/utils.py

```diff
@@ -353,7 +353,7 @@ def get_clusters():
   try:
     return instance.run_sync(lambda: API.get_clusters())
   except Exception as ex:
-    Log.error('Error: %s' % str(ex))
+    Log.error(str(ex))
```

Comment:
For a Python program ``test.py``:
````
def a():
  return 1 / 0

def b():
  return a()

def c():
  return b()

try:
  c()
except Exception as ex:
  print(str(ex))
  pass
````

``python test.py`` will only print out "division by zero"

An informative error will be printed out after ``Log.error(str(ex))`` (I can add cluster name, though)